modules/gapi/include/opencv2/gapi/infer/ie.hpp

```diff
@@ -196,70 +196,104 @@ template<typename Net> class Params {
     detail::ParamDesc desc;
 };
 
+/*
+* @brief This structure provides functions for generic network type that
+* fill inference parameters.
+* @see struct Generic
```

Comment:
Is it really `@see` or `@sa` ?

Reply:
https://www.doxygen.nl/manual/commands.html#cmdsee

```
\see { references }
Equivalent to \sa. Introduced for compatibility with Javadoc.
```